dev_requirements

```diff
@@ -4,12 +4,14 @@ pytest-cov
 pytest-env
 uamqp~=1.2
 responses
-urllib3[secure]>=1.21.1,<=1.25
+urllib3[secure]>=1.26.5
 black
 wheel==0.30.0
 pre-commit
 pylint
 flake8
 azure-storage-blob
 pytest-xdist
-pytest-rerunfailures
\ No newline at end of file
+pytest-rerunfailures
+tqdm
```

Comment:
`tqdm` and `jsonschema` should be installed with the extension and are not `dev_requirements` items.

Reply:
correct. I fixed it